sky/resources.py

```diff
@@ -1014,17 +1014,19 @@ def _try_validate_labels(self) -> None:
                     'The following labels are invalid:'
                     '\n\t' + invalid_table.get_string().replace('\n', '\n\t'))
 
-    def get_cost(self, seconds: float) -> float:
-        """Returns cost in USD for the runtime in seconds."""
-        hours = seconds / 3600
+    def get_hourly_cost(self) -> float:
         # Instance.
         hourly_cost = self.cloud.instance_type_to_hourly_cost(
             self._instance_type, self.use_spot, self._region, self._zone)
         # Accelerators (if any).
         if self.accelerators is not None:
             hourly_cost += self.cloud.accelerators_to_hourly_cost(
                 self.accelerators, self.use_spot, self._region, self._zone)
-        return hourly_cost * hours
+        return hourly_cost
+
+    def get_cost(self, seconds: float) -> float:
+        """Returns cost in USD for the runtime in seconds."""
+        return self.get_hourly_cost() * seconds / 3600
```

Comment:
Do we need to add a new API here? Can we just use `get_cost(3600)` in the runpod cloud to get the hourly cost?

Reply:
removed